src/internal/connector/exchange/service_restore.go

```diff
@@ -362,14 +345,15 @@ func RestoreExchangeDataCollections(
 		}
 	}
 
-	status := support.CreateStatus(ctx,
+	status := support.CreateStatus(
+		ctx,
 		support.Restore,
 		len(dcs),
 		metrics,
-		errs,
+		errs.Err(),
```

Comment:
since this only returns the first error and restore operation currently gets error information through the `AwaitStatus()` call isn't this going to hide other errors that were encountered for this particular collection?

Reply:
Exposing the other errors relies on the controller reviewing errs.Errs().  Nothing here is lost, ultimately.  First priority is to ensure we're tracking whether or not an error happened, and for that we only need to find one.  The long term goal is to remove error tracking from support.Status altogether.